CQA Installing Proxy guide (#4932)
theforeman/foreman-documentation: guides/common/assembly_preparing-environment-for-capsule-installation.adoc

= Preparing your environment for installation

[role="_abstract"]

> Before installing {SmartProxyServer}, ensure your environment meets the operating system requirements, system requirements, storage requirements, and network port configurations to enable successful installation and operation.

Reviewer: Shorter, leads with the benefit, and avoids feature-centric language (enable).

```suggestion
For a successful installation and operation of {SmartProxyServer}, ensure your environment meets the operating system requirements, system requirements, storage requirements, and network port configurations.
```
